extensions/ql-vscode/README.md

```diff
@@ -49,7 +49,7 @@ If you have any difficulty with CodeQL CLI access, see the **CodeQL Extension Lo
 ### Cloning the CodeQL starter workspace
 
 When you're working with CodeQL, you need access to the standard CodeQL libraries and queries.
-Initially, we recommend that you clone and use the ready-to-use starter workspace, https://github.com/github/vscode-codeql-starter/.
+Initially, we recommend that you clone and use the ready-to-use starter workspace, [https://github.com/github/vscode-codeql-starter/].
```

Comment:
What's the reasoning behind the square brackets? They aren't rendered in the markdown preview 🤔 

If you want to avoid the plain text URL, you could use "starter workspace" as the link text: 

```suggestion
Initially, we recommend that you clone and use the ready-to-use [starter workspace](https://github.com/github/vscode-codeql-starter/).
```